src/plugin_system/schema_manager.py

```
@@ -354,6 +367,46 @@ def validate_config_against_schema(self, config: Dict[str, Any], schema: Dict[st
+            return schema
+
+        enhanced = copy.deepcopy(schema)
+        enhanced.setdefault("properties", {})
+        if "skin" not in enhanced["properties"]:
+            names = {sid: (matching.get(sid, {}).get("name") or sid) for sid in choices}
+            enhanced["properties"]["skin"] = {
+                "type": "string",
+                "title": "Visual Skin",
+                "description": "Replace this scoreboard's look with an installed skin "
+                               "(data, scheduling, and vegas mode are unaffected)",
+                "enum": ["built-in"] + choices,
```

Ruff (0.15.21)

[warning] Consider `["built-in", *choices]` instead of concatenation

Replace with `["built-in", *choices]`

(RUF005)